grants/accepted_grant_applications.md

```diff
@@ -1,8 +1,8 @@
 # Accepted Grant Applications [WIP!] <!-- omit in toc -->
 
-> **:loudspeaker:** This page gives an overview of accepted grants, their progress and a link to their GitHubs. **You should be aware that the grant application itself is often an independent project, that is unrelated to other work done by the teams. Also there is a clear difference between accepting a grant application and the successful delivery of a grant.** 
+> **:loudspeaker:** This page provides an overview of accepted grants, their progress and a link to their GitHub repositories. In cases where the link points to an organization, **you should be aware that the grant application itself is often an independent project unrelated to other work done by the teams**. Besides, **there is a clear difference between an application being accepted and the project being successfully delivered.** Only projects that have successfully delivered a milestone are allowed to make public announcements on the matter or to use our [badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md) for grant recipients. Violations to this policy can be reported [here]().
```

Comment:
This would be a good place to make this clear, in case we want to. Don't know what the right place to report violations would be.

Reply:
Good idea. I suggest using grants@web3.foundation

Reply:
Whichever you prefer

```suggestion
> **:loudspeaker:** This page provides an overview of accepted grant applications, their progress and a link to their GitHub repositories. In cases where the link points to an organization, **you should be aware that the grant application itself is often an independent project unrelated to other work done by the teams**. Besides, **there is a clear difference between accepting an application and the successful delivery of the project.** Only projects that have successfully delivered a milestone are allowed to make public announcements on the matter or to use our [badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md) for grant recipients. Violations to this policy can be reported [here]().
```

Reply:
Whatever you prefer

Reply:
Give me a minute